### ISSUE TITLE
Reduce lib size

### ISSUE THREAD

**Issue by icebob** [leo/args#120]
Hi,

I checked your lib size because it is~300kB: https://arve0.github.io/npm-download-size/#args
But your code is very small (1kB), but you have "string-similarity" dependency which is 300kB. 86% of the whole size.

Do you plan to change it to other smaller lib?

Thanks!

**Comment by ntwcklng** [comment]
Hey @icebob,

`args` is a CLI which only runs on Desktops, so in my opinion the size does not matter that much as on mobile web apps. But in general i like smaller and lighter modules.
Do you know a smaller module like `string-similarity`? 
`string-similarity` depends on several `lodash` functions (=99% of the size), so maybe we can create a PR without `lodash`.

**Comment by icebob** [comment]
I found [leven](https://github.com/sindresorhus/leven). It's only 2.3kB, but it's a Levenshtein distance algorithm.

**Comment by ntwcklng** [comment]
Okay, do you want to create a PR?

**Comment by icebob** [comment]
Yes, I will.

**Comment by ntwcklng** [comment]
Thanks! Maybe we should wait until #116 is merged?!

**Comment by ntwcklng** [comment]
Done 👍

**Comment by icebob** [comment]
Great! I'm trying to change similarity lib...

**Comment by icebob** [comment]
I'm trying, but how can I test it? I see there is no test case for similarity and I don't know when this code is called. Could you help me?

**Comment by ntwcklng** [comment]
You can create a file, where you require your local `args` module like this:

`similarity.js`
```js
#!/usr/bin/env node
const args = require('../args/lib/')

args
    .option('t', 'This is a test', 'foo')
    .option('verbose', 'verbose option', 'jep')

const flags = args.parse(process.argv, {
  version: false,
  mri: {
    string: 't'
  }
})
console.log(flags)
```
switch to that dir and run `node similarity.js` and test some typos: `node similarity.js --ferbose`, `node similarity.js --vernose`

**Comment by icebob** [comment]
I'm done. Could you check it? #121

**Comment by icebob** [comment]
@ntwcklng When can you check & merge it?

**Comment by ntwcklng** [comment]
I will look at it next week 👍

**Comment by ntwcklng** [comment]
Released in 4.0.0, thanks @icebob 😎

**Comment by icebob** [comment]
Great! Me too! :sunglasses:
